workspaces/mujoco_workspace.py

```diff
@@ -77,16 +81,28 @@ def train(self):
                 self.save_snapshot()
 
             if done:
+                values = np.array(val_seq)
+                ret = np.array(rew)
+                for i in range(len(rew)):
+                    s = 0
+                    for j in range(len(rew) - i):
+                        s += rew[j]*self.agent.gamma**j
+                    ret[i] = s
                 self._train_episode += 1
-                print("Episode: {}, total numsteps: {}, return: {}".format(self._train_episode, self._train_step, round(info["episode"]["r"], 2)))
+                print("Episode: {}, total numsteps: {}, return: {}".format(self._train_episode, self._train_step, round(info["episode"]["r"][0], 2)))
```

Comment:
I do not see where you changed `info` to contain arrays, did you forget to commit this part?